uniffi_bindgen/src/bindings/kotlin/gen_kotlin/enum_.rs

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
One thing I like about this re-arrangement of the code, is that the docstring of this module would be a great place to put the narrative docs about how enums work in the Kotlin. I like having an obvious place for such docs.